includes/variables.php

```diff
@@ -76,6 +76,15 @@
     }
 }
 
+//define entity fields (stored in option tsml_entity)
+$tsml_entity_fields = [
+    'entity',
+    'entity_email',
+    'entity_phone',
+    'entity_location',
+    'entity_url',
+];
```

Comment:
do we need to add feedback email here? the idea is that they should pass through from the imported service entity